.travis.yml

```diff
@@ -0,0 +1,19 @@
+if: branch = master and type IN (pull_request)
+language: cpp
+services:
+  - docker
+env:
+before_install: true
+install: true
+before_script: true
+script: true
+jobs:
+  include:
+    - stage: optional
```

Comment:
I don't like the "optional" literal too much... I doesn't provide clues about what is about.

What about "compliance" ?

Reply:
Good idea)

Reply:
fixed in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/85f9adb0d02f1a9d1906bea2198af60546f9b612